host/tools/build/Microsoft.Azure.Functions.DotnetIsolatedNativeHost.nuspec

```diff
@@ -4,7 +4,7 @@
     <id>Microsoft.Azure.Functions.DotNetIsolatedNativeHost</id>
     <title>Microsoft Azure Functions dotnet-isolated native host</title>
     <tags>dotnet-isolated azure-functions azure</tags>
-    <version>1.0.0-preview8</version>
+    <version>1.0.0-preview805</version>
```

Comment:
Need to fix?

Reply:
I kept using different series for different experiments. When I decided to use net8 for AOT compilation, I switched to the 800 series. and this is the final version. I kept it as 805 because that is what we merged to host (the DotNetIsolatedNativeHost package version).